build.sbt

```diff
@@ -32,7 +32,7 @@ lazy val commonSettings = Seq(
   scalaVersion := default_scala_version,
   crossScalaVersions := all_scala_versions,
   fork := true,
-  scalacOptions ++= Seq("-target:jvm-1.8"),
+  scalacOptions ++= Seq("-target:jvm-1.8", "-Ywarn-unused:imports"),
```

Comment:
does this fail the build if unused imports are there?

Reply:
No, `warn` only.